com.stansassets.build/Editor/Steps/IBuildContext.cs

```diff
@@ -21,5 +21,12 @@ public interface IBuildContext
         /// Key value
         /// </summary>
         string BuildAlias { get; }
+
+
+        /// <summary>
+        /// Set scenes path what be included to build
```

Comment:
Important to say here that SetScenes works only in case when a user sets scenes **BEFORE** UnityPlayerBuildStep (has priority lower than 0), otherwise it just changes paths collection for next steps.
Maybe something like:
_"Sets scene paths for a build. Will affect build only in case if called before UnityPlayerBuildStep (has priority lower than 0)"_

Reply:
updated